lib/components/DateTime/DatePicker.module.scss

```diff
@@ -114,11 +102,12 @@ $date-picker-input-height: 8*$grid-size;
         right: 0;
     }
 
-    background-color: var(--color-bg-input-rest);
-    border: 1px solid var(--color-border-selected);
+    background-color: var(--color-content-background-primary);
+    border: 1px solid var(--color-accent-hover);
+    
     :global(.md-dropdown-container) & {
-        background-color: var(--color-bg-input-rest);
-        border: 1px solid var(--color-border-selected);
+        background-color: var(--color-content-background-primary);
+        border: 1px solid var(--color-accent-hover);

```

Comment:
why hover instead of selected?

Reply:
Actually both are wrong... this should be `--color-foreground-secondary` to have consistent border colors for all the callout fields (dropodown, combo and this)

Reply:
is `accent-hover` to be consistent with what we do with other fields and with what fabric does with dropdowns/callout components